src/domain/class/Car.js

```diff
@@ -0,0 +1,33 @@
+class CarPosition {
+  #name;
+  #position;
+
+  constructor(name) {
+    this.#name = name;
+    this.#position = [0];
```

Comment:
배열이니까 변수명을 복수형으로 만들어보면 좋을 것 같아요.
```suggestion
    this.#positions = [0];
```